src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct.txt

```diff
@@ -0,0 +1,5 @@
+>> CountRows(Distinct([{a: 1}, {a: 1}, {a: 3}], Value.a))
+2
+
+>> Distinct([{a: 1}, {a: 1}, {a: 3}], Value.a)
+Table({Result:1},{Result:3})
```

Comment:
>Result [](http://example.com/codeflow?start=7&length=6)

fyi - we'll trying to standardize on using "Value"  name, so we may end up changing this to be Value instead of Result.

Reply:
Talked offline. Leaving value for now, easy to change later.